backend/api/Areas/Reports/Mapping/Property/PropertyMap.cs

```diff
@@ -39,7 +45,8 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.RentableArea, src => src.RentableArea)
                 .Map(dest => dest.OccupantName, src => src.OccupantName)
                 .Map(dest => dest.LeaseExpiry, src => src.LeaseExpiry)
-                .Map(dest => dest.TransferLeaseOnSale, src => src.TransferLeaseOnSale);
+                .Map(dest => dest.TransferLeaseOnSale, src => src.TransferLeaseOnSale)
+                .Map(dest => dest.ProjectNumbers, src => src.ProjectNumbers);
```

Comment:
If you're not deserializing it probably don't need to include the `_serializerOptions`